Gemfile.lock

```diff
@@ -34,43 +34,43 @@ GEM
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.2)
+    i18n (1.8.3)
```

Comment:
### [i18n](https://github.com/ruby-i18n/i18n)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.8.2 → 1.8.3

[_[change-log](https://github.com/svenfuchs/i18n/releases), [source-code](https://github.com/svenfuchs/i18n)_]





<details>
<summary>Commits</summary>

A change of **35** commits. See the full changes on [the compare page](https://github.com/ruby-i18n/i18n/compare/v1.8.2...v1.8.3).

These are the first 10 commits:
- (04e6f82) [add option to disable fallbacks for I18n.exists? check](https://github.com/ruby-i18n/i18n/commit/04e6f823f7f76be8f386bc8a8dc3bbcb952c39b5)
- (d9dfd86) [Add activesupport's Hash#deep_merge refinement](https://github.com/ruby-i18n/i18n/commit/d9dfd86c5b51897643846fd8591f17b2a5aedf29)
- (741add9) [Revert "Revert "Chain fallback backends""](https://github.com/ruby-i18n/i18n/commit/741add9dba3fe8189454f47c5774b45211ab11a9)
- (aef8596) [Merge pull request #509 from vipera/fix-chain-fallback-backe…](https://github.com/ruby-i18n/i18n/commit/aef8596111fc0ae3c70c71b7f88be58a22a659f4)
- (6d18906) [Update issue templates](https://github.com/ruby-i18n/i18n/commit/6d18906df97d9268239831341fbd4e5c7890e222)
- (1799e4e) [Switch to new GitHub issue template](https://github.com/ruby-i18n/i18n/commit/1799e4e0c306e6d81b6c5bf3a69513e30228bff9)
- (eac7a9e) [Merge pull request #482 from jeffjyang/add-i18n-exist-check-…](https://github.com/ruby-i18n/i18n/commit/eac7a9e6bb481c3c325fb030b9f57534da66da09)
- (4336047) [Add the test to fail](https://github.com/ruby-i18n/i18n/commit/43360471a5fd2f85a2da6e845854387c6c098425)
- (572bca8) [Add reject to remove attributes from entry](https://github.com/ruby-i18n/i18n/commit/572bca8b5eb55178eed622ed6befb9512017bf4d)
- (d07cb48) [Added double splat to resolve deprecation warning](https://github.com/ruby-i18n/i18n/commit/d07cb485eeac072722dc054664c836614fa85ae3)


</details>